blog/_posts/2021-10-21-scala-3.1.0-released.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.0 released!
+---
+Hello from the Scala 3 team! It has already been six weeks since we have announced the release candidate for the first minor version after the initial release of Scala 3. Now, after two more RCs, we can confidently promote Scala 3.1.0-RC3 to a new stable release of the language. In other words, Scala 3.1.0 is officially out!
+
+## Compatibility notice
+
+This is a first *minor* release after the initial release of Scala 3.0. This has following consequences:
+
+- Scala 3.1 is backward binary compatible: you can use dependencies compiled with Scala 3.0 in 3.1 projects.
+- Scala 3.1 is _not_ forward binary compatible: you _cannot_ use dependencies compiled with Scala 3.1 in Scala 3.0 projects.
+- Updating Scala to 3.1 is not a forward compatible change. If you are a library maintainer and are following SemVer you may want to update in your next minor release.
```

Comment:
The messaging in this bullet feels different than the messaging below; the message here has a "update, but bump your minor version" feel, whereas the wording at the end of the section has more of a "don't update".

The two wordings aren't outright contradictory, exactly, but taken together, I find them a bit confusing and I think other readers will as well.

It's not the only possible solution, but I think I would suggest omitting this bullet and saying whatever needs to be said about this in the paragraph at the end of the section.